src/plugins/kibana/public/visualize/styles/main.less

```diff
@@ -2,21 +2,247 @@
 @import (reference) "~bootstrap/less/list-group";
 @import (reference) "~ui/styles/list-group-menu";
 
+@icon-background-color: #416376;
+
+ul.li-not-striped {
+  li:nth-child(odd) {
+    background-color: transparent !important;
+  }
+
+  li:nth-child(even) {
+    background-color: transparent !important;
+  }
+
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+}
+
+ul.li-striped {
+  li {
+    border: none;
+  }
+
+  li:nth-child(odd) {
+    background-color: @kibanaGray6;
+  }
+
+  li:nth-child(even) {
+    background-color: @white;
+  }
+}
+
+.step-1-wizard {
+  flex: 1;
+  background-color: @kibanaGray6;
+}
+
+.step-2-list {
+  border: none;
+  background-color: @kibanaGray6;
+  border-radius: 0;
+  padding-left: 5px;
+}
+
+.search-label {
+  padding-left: 5px;
+}
+
+.list-search-item {
+  border: none;
+  background-color: @kibanaGray6;
+  border-radius: 0;
+  padding: 0;
+}
+
+.saved-search-label {
+  padding-left: 10px;
+
+  label {
+    margin-bottom: 0;
+  }
+}
+
+.paginate-style {
+  padding-bottom: 1px;
+  background-color: @kibanaGray6;
+}
+
+.form-text {
```

Comment:
Is form-text blue anywhere else, is there a way we can apply this more generically?